script_stage/scriptJobConfig.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I'd rather see this as a job-dsl

There's a few things in here that don't really make sense (environment specific)